### PR TITLE
Lambert conformal secant latitudes

### PR DESCRIPTION
**Pull request by bjlittle**
This PR addresses issue #3228

### CODE REVIEW

File: lib/iris/tests/test_coordsystem.py [hunk 1]
```diff
@@ -375,6 +375,30 @@ def test_as_cartopy_projection(self):
         self.assertEqual(res, expected)
 
 class Test_LambertConformal(tests.GraphicsTest):
+    def test_fail_secant_latitudes_none(self):
+        emsg = 'one or two secant latitudes required'
+        with self.assertRaisesRegexp(ValueError, emsg):
+            LambertConformal(secant_latitudes=())
```
[the remark this answers is not in the document]
Comment (bjlittle) [reply lib/iris/tests/test_coordsystem.py]: *sigh*...

**Comment by bjlittle** [comment]
@corinnebosley This PR will fail and require to be rebased once https://github.com/SciTools/iris/issues/3230 is resolved

File: requirements/core.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@
 cartopy
 #conda: proj4<5
 cf_units>=2
-cftime
+cftime!=1.0.2.1
```
Comment (bjlittle) [inline requirements/core.txt]: This pin was merged onto `master` by #3215 ... adding it hear for the `2.2.x` branch...

**Comment by corinnebosley** [comment]
@bjlittle Looks good to me. I'm pushing the button.

**Comment by bjlittle** [comment]
Awesome, thanks @corinnebosley :smile:

**Pull request merged**